packages/docs/getting-started/running-a-validator-in-baklava.md

```diff
@@ -558,8 +558,186 @@ celocli validator:show $CELO_VALIDATOR_RG_ADDRESS
 
 ### Running the Attestation Service
 
-{% hint style="warning" %}
-**Under construction** We are currently working on updating the attestation service guide to reflect changes in workflow for `ReleaseGold`.
+As part of the [lightweight identity protocol](/celo-codebase/protocol/identity), Validators are expected to run an [Attestation Service](https://github.com/celo-org/celo-monorepo/tree/master/packages/attestation-service) to provide attestations that allow users to map their phone number to an account on Celo. Be sure to allow TCP connections to your Attestations machine on port 80 for all IP addresses.
+
+Just like with the Validator signer, we'll want to authorize a separate Attestation signer. For that let's start our node on the Attestations machine:
+
+```bash
+# On the Attestation machine
+# Note that you have to export CELO_IMAGE, NETWORK_ID and CELO_VALIDATOR_ADDRESS on this machine
+export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:baklava
+export NETWORK_ID=40120
+export CELO_VALIDATOR_RG_ADDRESS=<YOUR-CELO-VALIDATOR-RG-ADDRESS>
+mkdir celo-attestations-node
+cd celo-attestations-node
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE init /celo/genesis.json
+export BOOTNODE_ENODES=`docker run --rm --entrypoint cat $CELO_IMAGE /celo/bootnodes`
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE account new
+export CELO_ATTESTATION_SIGNER_ADDRESS=<YOUR-ATTESTATION-SIGNER-ADDRESS>
+```
+
+Let's generate the proof-of-possession for the attestation signer:
+
+```bash
+# On the Attestation machine
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE --nousb account proof-of-possession $CELO_ATTESTATION_SIGNER_ADDRESS $CELO_VALIDATOR_RG_ADDRESS
+```
+
+With this proof, authorize the attestation signer on your local machine:
+
+```bash
+# On your local machine
+export CELO_ATTESTATION_SIGNER_SIGNATURE=<YOUR-ATTESTATION-SIGNER-SIGNATURE>
+export CELO_ATTESTATION_SIGNER_ADDRESS=<YOUR-ATTESTATION-SIGNER-ADDRESS>
+celocli releasegold:authorize --from $CELO_VALIDATOR_RG_ADDRESS --role attestation --signature 0x$CELO_ATTESTATION_SIGNER_SIGNATURE --signer $CELO_ATTESTATION_SIGNER_ADDRESS
```

Comment:
I haven't had to add `0x` when I've run these commands, but if you've actually run these commands recently then I'll defer to the most current version

Reply:
this should also be `celocli releasegold:authorize --contract $CELO_VALIDATOR_RG_ADDRESS ...` emphasis on *--contract*